Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj

```diff
@@ -37,7 +37,7 @@
       <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
     </Content>
-    <Content Include="assemblyMetadata.json" />
+    <EmbeddedResource Include="assemblyMetadata.json" />
```

Comment:
Similar for changes here, I would have expected this embedded resource change to go in with ODS-4664, but I don't see any of the actual changes to embed the xsds?

Reply:
I need <EmbeddedResource Include="assemblyMetadata.json" /> from ODS-4664 to make sure that to create package from 5.1.0 tag . Note :   I created new branch from v5.1.0 tag which didn't have  ODS-4664 code changes . So I have to add it here also in this branch . Thank you !